py/server/tests/test_parquet.py

```diff
@@ -183,7 +179,40 @@ def test_round_trip_data(self):
             # "nullBigDecColumn = (java.math.BigDecimal)null",
             # "nullBigIntColumn = (java.math.BigInteger)null"
         ])
+        return dh_table
+
+    def get_table_with_array_data(self):
+        # create a table with columns to test different types and edge cases
+        dh_table = empty_table(20).update(formulas=[
+            "someStringArrayColumn = new String[] {i % 10 == 0?null:(`` + (i % 101))}",
```

Comment:
Is there a way to convert vector columns into arrays instead of doing the way I have done?